test.py

```diff
@@ -115,27 +115,64 @@ def features_in_tile_layer(z, x, y, layer):
         raise Exception, "Tile %r: %s" % (url, e.message), sys.exc_info()[2]
 
 
+def count_matching(features, properties):
+    """
+    Returns a tuple containing the total number of features in the argument
+    `features` and the number of features matching the properties.
+    """
+
+    num_features = len(features)
+    num_matching = 0
+
+    for f in features:
+        if match_properties(f['properties'], properties):
+            num_matching += 1
+
+    return (num_features, num_matching)
```

Comment:
Oh, cool. I didn't realise that. I find this more readable, but perhaps that's just unfamiliarity with Python.